ckanext/switzerland/helpers/frontend_helpers.py

```diff
@@ -26,6 +28,8 @@
     ogdch_term_utils.TERMS_OF_USE_BY_ASK: '#terms_by_ask',
 }
 
+DATE_PICKER_FORMAT = '%d.%m.%Y'
```

Comment:
Have you considered defining the `DATE_PICKER_FORMAT` once in the .ini-files? That way we would only have to define it once and only fetch it from the .ini-file here.

Reply:
that's a great idea, thanks!